packages/asc-web-common/store/AuthStore.js

```diff
@@ -154,6 +156,22 @@ class AuthStore {
     }
   };
 
+  thirdPartyLogin = async (SerializedProfile) => {
```

Comment:
Rename thirdPartyLogin to loginThirdParty

Rename SerializedProfile to profile